homework/Vasili_Zasinets/UserLoginTask/src/test/java/com/tutrit/java/quickstart/homework/UserLoginServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.tutrit.java.quickstart.homework;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class UserLoginServiceTest {
+    User userOne = new User("Login", "256");
+    UserLoginService userLoginService = new UserLoginService();
+    Logger log = LoggerFactory.getLogger("UserTest");
+
+    @Test
+    public void login() {
+
+        assertTrue(userLoginService.login(userOne, "256"));
+
+        log.info("User status - isBlock {}, Number of attempts {} ", userOne.isBlock, userOne.getNumberOfAttempts());
+
+        for (int i = 0; i < 3; i++) {
+            userLoginService.login(userOne, "456");
+            log.info("User status - isBlock {}, Number of attempts {} ", userOne.isBlock, userOne.getNumberOfAttempts());
+        }
```

Comment:
this is another test, where for (int i = 0; i < 3; i++) {
is a test conditions

Reply:
anyway, using business logic in tests is a bad practice. In unit test you need to define state of the program, one(!) input that should change that state and expected new state. 

that is why they are UNIT tests.